### PR TITLE
551 revert capacity planning power management

### PR DESCRIPTION
**Pull request by brandonpille**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
           go-version: 1.18
 
       - name: run docker image
-        run: docker run -d -p 9944:9944 threefolddev/tfchain-dev:2.2.0-rc5 --dev --ws-external
+        run: docker run -d -p 9944:9944 dylanverstraete/tfchain:2.2.0-rc3 --dev --ws-external
```
Comment (brandonpille) [inline .github/workflows/test.yml:18]: Not sure if this is correct?

Comment (brandonpille) [reply .github/workflows/test.yml:18]: No it's not. We need a new build once tfchain revert has been completed.

Comment (DylanVerstraete) [reply .github/workflows/test.yml:18]: I will create a docker image and give you the tag

**Comment by brandonpille** [comment]
@DylanVerstraete, @muhamadazmy can you guys review it one more time? Thanks :)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
           go-version: 1.18
 
       - name: run docker image
-        run: docker run -d -p 9944:9944 dylanverstraete/tfchain:2.2.0-rc3 --dev --ws-external
+        run: docker run -d -p 9944:9944 threefolddev/tfchain-dev:2.2.0-rc5 --dev --ws-external
```
Comment (xmonader) [inline .github/workflows/test.yml:18]: let's make sure this tag is updated

**Review by xmonader — DISMISSED** [review]

**Review by xmonader — CHANGES_REQUESTED** [review]
please make sure to rebase your branch and update the docker tag

**Comment by brandonpille** [comment]
> please make sure to rebase your branch and update the docker tag

Done. Can you review the changes please?

**Review by DylanVerstraete — APPROVED** [review]
lgtm

File: node.go [hunk 3]
```diff
@@ -464,7 +464,8 @@ func (s *Substrate) UpdateNode(identity Identity, node Node) (uint32, error) {
 		return 0, errors.Wrap(err, "failed to create call")
 	}
 
-	if callResponse, err := s.Call(cl, meta, identity, c); err != nil {
+	callResponse, err := s.Call(cl, meta, identity, c)
+	if err != nil {
 		return 0, errors.Wrap(err, "failed to update node")
 	} else {
```
Comment (xmonader) [inline node.go:470]: we already returned, no need to have an else block

File: contract.go [hunk 4]
```diff
@@ -545,7 +545,7 @@ func (s *Substrate) Report(identity Identity, consumptions []NruConsumption) (ha
 
 	callResponse, err := s.Call(cl, meta, identity, c)
 	if err != nil {
-		return hash, errors.Wrap(err, "failed to create report")
+		return callResponse.Hash, errors.Wrap(err, "failed to create report")
```
Comment (xmonader) [inline contract.go:548]: i think we used named returns here in the function for no reason

**Review by xmonader — CHANGES_REQUESTED** [review]

**Comment by brandonpille** [comment]
Thanks @xmonader. I made the changes, can you review them one more time please?

**Review by xmonader — APPROVED** [review]

**Pull request merged**